MapPointMapper/ViewController.swift

```diff
@@ -108,14 +109,31 @@ class ViewController: NSViewController, MKMapViewDelegate, NSTextFieldDelegate {
     renderer.strokeColor = colorWell.color
     return renderer
   }
-  
-  // MARK: NSTextFieldDelegate
-  override func keyUp(theEvent: NSEvent) {
```

Comment:
I don't think it should be necessary with the currently controller outlets triggering their subsequent actions.  Would this provide something I'm missing?